.github/workflows/ephemeral-env-cleanup.yml

```diff
@@ -25,3 +26,18 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: Authenticate to Google Cloud
+        id: auth
+        uses: google-github-actions/auth@v1
+        with:
+          token_format: access_token
+          credentials_json: ${{ secrets.GCP_SERVICE_CREDENTIALS_JSON }}
+          service_account: ${{ secrets.GCP_SERVICE_ACCOUNT_EMAIL }}
+      - name: Destroy Cloud Run service
+        run: |
+          environments=$(gcloud run services list --format json | jq '.[] | .metadata.name' -r)
+          for env in $environments; do
+            if [[ $env == websight-cms-* ]]; then
+              gcloud run services delete -q $env --region=europe-west1
```

Comment:
Shouldn't we handle errors somehow?
E.g. if there are 3 environments: "websight-cms-1", "websight-cms-2", "websight-cms-3" and deleting the first one fails, the pipeline will end up with error, but all the environments will remain in GCP.

Reply:
When I tested this workflow, it looked that delete is asynchronous operation.